src/theme/components/headerMobileNav.ts

```diff
@@ -0,0 +1,59 @@
+const HeaderMobileNav = {
+  parts: ["bottomLinks", "sideNav"],
+  baseStyle: () => ({
+    whiteSpace: "initial",
+    position: "absolute",
+    left: "0px",
+    backgroundColor: "ui.black",
+    width: "100%",
+    zIndex: "99999",
+    color: "ui.white",
+    sideNav: {
+      textAlign: "right",
+      padding: "15px",
+      marginBottom: "0",
+      "li:not(:first-child)": {
+        marginTop: "10px",
+      },
+      a: {
+        color: "ui.white",
+        textDecoration: "none",
+        _hover: {
+          color: "ui.white",
+          textDecoration: "none",
+        },
+      },
+    },
+    bottomLinks: {
+      display: "grid",
```

Comment:
This is what I meant regarding that `SimpleGrid` seemed to have lost this rule when adding the `__css` prop.